src/electron/views/device-connect-view/components/android-setup/prompt-choose-device-step.tsx

```diff
@@ -26,33 +50,42 @@ export const PromptChooseDeviceStep = NamedFC<CommonAndroidSetupStepProps>(
             console.log(`androidSetupActionCreator.rescan()`);
         };
 
-        const items = [
+        // Available devices will be retrieved from store in future feature work
+        const devices: DeviceMetadata[] = [
             {
-                value: 'Phone 1',
+                description: 'Phone 1',
```

Comment:
DeviceMetadata will eventually need to include the id field that will be used to select the target device. Should that be in this PR or in a future PR?

Reply:
I think a future PR makes sense because the ID won't show up on the UI & this component doesn't consume the store data yet